apps/org/views.py

```diff
@@ -56,3 +59,27 @@ def get_queryset(self):
         """A user may only delete Organizations that they are associated with."""
         qs = super().get_queryset()
         return qs.filter(users=self.request.user)
+
+
+class LocalUserAPI(LoginRequiredMixin, View):
+    ''' Setting up a local endpoint that talks to the VMI endpoint and filters to
+        only include users with user social auth uids that match '''
+    def get(self, request, *args, **kwargs):
+        response = get_vmi_user_data(request)
+        user_data = []
+
+        if isinstance(response.json(), list):
+            for i, user in enumerate(response.json(), 0):
```

Comment:
This is really interesting, it limits the search to accounts that are already active in smh_app.

@aviars what's the desired behavior if, as an organizational user, I search for and want to access someone who somehow has an identity on VMI but isn't set up in the smh_app?

Reply:
it's been decided that the first version of this will search only within the smh app. The ability to search against vmi can be added later as the desired behavior is defined.

I think continuing to use this place as the interface for search makes sense, as it allows us to change the implementation of the search without changing the external facing behavior.